src/protobuf-net.Reflection.Test/protobuf-net.Reflection.Test.csproj

```diff
@@ -16,6 +16,9 @@
     <None Update="macosx\protoc">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Schemas\issues\issue833.proto">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
This shouldn't be needed - already handled by the wildcard further below? Otherwise: thanks, looks good

Reply:
Ups, removed. Thanks for the quick feedback.